sdk/lib/3rdparty/freetype/src/psaux/psintrp.c

```diff
@@ -3029,6 +3036,10 @@
     cf2_arrstack_finalize( &subrStack );
     cf2_stack_free( opStack );
 
+#ifdef __REACTOS__
+    free(&glyphPath);
+#endif
```

Comment:
`#undef glyphPath` would be good here

Reply:
Yeap, this is just a quick patch, and it only made its way here because of the madness in https://build.reactos.org/builders/Build%20GCCLin_x86/builds/20140/steps/preparing%20source/logs/stdio :)